tests/test_toolbox.py

```diff
@@ -601,6 +614,19 @@ def test_hypot(self):
             self.assertEqual(tb.hypot(numpy.array([3.,4.])), 5.0)
             self.assertEqual(tb.hypot(numpy.array([3,4])), 5.0)
 
+    def test_hypot_array(self):
+        """hypot when called on an array falls to C"""
```

Comment:
How do these tests verify that this falls through to C?

Reply:
There is no way to verify it that I know of, but there we no tests that could go to C as there were no arrays.

Reply:
I read the docstring as "test that hypot falls to C" and that's not really what it's testing..."hypot on array, may fall to C" perhaps?

Reply:
This whole function is primarily a copy of test_hypot; really only the fop set of invals changes (to be numpy arrays), and there was already a test with numpy arrays at the bottom of test_hypot. (Which, until you fixed, didn't actually fall to C, so it didn't show that bug, but that's something that I guess would probably be caught with coverage.) What we need to do is test with those inputs both in C and Python....you want me to just do that work?